addons/website_payment/static/src/js/payment_form.js

```diff
@@ -62,30 +62,29 @@ PaymentForm.include({
                 'email': _t('Email'),
                 'country_id': _t('Country'),
             };
+            this.el.querySelectorAll("div.invalid-feedback").forEach((el) => el.remove());
+            this.el.querySelectorAll(".is-invalid").forEach((el) => el.classList.remove("is-invalid"));
             for (const id in mandatoryFields) {
                 const fieldEl = this.el.querySelector(`input[name="${id}"],select[name="${id}"]`);
-                fieldEl.classList.remove('is-invalid');
```

Comment:
Why not keep this line here instead of re-selecting the `.is-invalid` elements before this loop?